wallet/createtx.go

```diff
@@ -34,6 +34,19 @@ import (
 // Constants and simple functions
 
 const (
+	// A worst case signature script to redeem a P2PKH output for a
```

Comment:
wrapping still inconsistent from line to line.  If you are using vscode, use this extention: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap
With this extension, you can just hit ALT+Q while your cursor is anywhere in the comment block to have it wrap correctly.